src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+  width: calendarWidth(
+    $day-size,
+    $colums-count,
+    $gap,
+    $padding
+  );
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    line-height: 100px;
+    text-align: center;
+    font-size: 30px;
+    border: 1px solid $color-black;
+    background-color: $color-white-grey;
+    transition: 0.5s;
+
+    @for $index from 1 through 31 {
+      &:nth-child(n + #{$index})::before {
+        content: "#{$index}";
+      }
+
+      &:hover {
+        cursor: pointer;
+        background-color: $color-pink;
+        transform: translateY(-20px);
+      }
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index}
+    :nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $num in $day-value {
+    &--start-day-#{$day} :first-child {
+      margin-left: $num * $day-size + $num * $gap;
```

Comment:
Wrong code, check it

![Screenshot 2023-02-09 at 20 05 35](https://user-images.githubusercontent.com/54940595/217900032-4ecefdc5-9a28-4a2d-a897-45e81a97561a.png)

Reply:
fixed